wiki/CreatingOutfits.md

```diff
@@ -38,7 +38,7 @@ Outfits work by modifying the attributes of your ship. Many of the attributes ar
 
 Most attributes are given as a single number, but there are a few "special" attributes:
 
-* `category`: which outfitter category to show this outfit in. The category must be one of the following if it is to be purchasable:
+* `category`: which outfitter category to show this outfit in. The outfit given must be a valid outfit category, which may be specified by a category `"outfit"` node **(v. 0.9.15)**. See [categories.txt](https://github.com/endless-sky/endless-sky/blob/master/data/categories.txt) for an example. The existing valid vanilla outfit categories are the following:
```

Comment:
I t4hink this might be a bit ambiguous as to what the version number ois referring to. Maybe something like:
```suggestion
* `category`: which outfitter category to show this outfit in. The outfit given must be a valid outfit category.
The list of available categories is specified by a `category "outfit"` node **(v. 0.9.15)**. See [categories.txt](https://github.com/endless-sky/endless-sky/blob/master/data/categories.txt) for an example.
The existing valid vanilla outfit categories are the following:
```
Or, do something similar to what you have in the CreatingShips file.